docs/software/python-cli/usage.mdx

```diff
@@ -29,19 +70,141 @@ INFO  | 18:38:04 711 [DeviceTelemetryModule] Sending packet to phone
 INFO  | 18:38:04 711 Telling client we have new packets 28
 ```
 
+It is worth noting that as long as the `meshtastic --noproto` is running in a terminal, you you might not be able to successfully "get" or "set" any device parameters as described in the following chapters.
+
 ## Getting a list of User Preferences
 
-You can get a list of user preferences by running '--get' with an invalid attribute such as 'all'.
+You can get a list of user preferences by running `--get` with an invalid (not known to meshtastic) attribute such as `all`.
 
 ```shell
 meshtastic --get all
 ```
 
-## Changing settings
+```shell title="A snippet of what you would see in the Resulting Output"
+# A snippet of what you would see in the Resulting Output:
+...
+
+power:
+    power.adc_multiplier_override
+    power.device_battery_ina_address
+    power.is_power_saving
+    power.ls_secs
+    power.min_wake_secs
+    power.on_battery_shutdown_after_secs
+    power.powermon_enables
+    power.sds_secs
+    power.wait_bluetooth_secs
+network:
+    network.address_mode
+    network.eth_enabled
+    network.ipv4_config
+    network.ntp_server
+    network.rsyslog_server
+    network.wifi_enabled
+    network.wifi_psk
+    network.wifi_ssid
+display:
+    display.auto_screen_carousel_secs
+    display.compass_north_top
+    display.compass_orientation
+    display.displaymode
+    display.flip_screen
+    display.gps_format
+    display.heading_bold
+    display.oled
+    display.screen_on_secs
+    display.units
+    display.wake_on_tap_or_motion
+
+...
+```
+
+## Getting a list of User Preferences including the Values they are currently set to
```

Comment:
This title is too long for an h2, h2's show up in the navbar on the side.

> Getting Preferences and Values

Should suffice